### PR TITLE
sgadmin: add ability to prompt for passwords + warn when cluster consists of nodes with different versions

### PR DESCRIPTION
**Pull request by floragunncom**
ask for passwords (SGCS-102)
warn when cluster consists of nodes with different versions (by Wernli)

### CODE REVIEW

File: src/main/java/com/floragunn/searchguard/tools/SearchGuardAdmin.java [hunk 1]
```diff
@@ -236,8 +237,17 @@ private static void main0(final String[] args) throws Exception {
...
-            tspass = line.getOptionValue("tspass", tspass); //TODO null? //when no passwd is set
+
+            promptForPassword = line.hasOption("prompt");
+            
+            if(kspass == null || kspass.isEmpty()) {
+                kspass = line.getOptionValue("kspass",promptForPassword?null:"changeit");
+            }
+            
+            if(tspass == null || tspass.isEmpty()) {
+                tspass = line.getOptionValue("tspass",promptForPassword?null:kspass);
```
Comment (floragunncom) [inline src/main/java/com/floragunn/searchguard/tools/SearchGuardAdmin.java]: either:

tspass = line.getOptionValue("tspass",promptForPassword?null:"changeit")

or 

tspass = line.getOptionValue("tspass",kspass)

Comment (jochenkressin) [inline src/main/java/com/floragunn/searchguard/tools/SearchGuardAdmin.java]: either:

tspass = line.getOptionValue("tspass",promptForPassword?null:"changeit")

or

tspass = line.getOptionValue("tspass",kspass)

Comment (floragunncom) [reply src/main/java/com/floragunn/searchguard/tools/SearchGuardAdmin.java]: why?

**Comment by floragunncom** [comment]
merged manually

**Pull request closed without merging**